packages/runtime-core/src/hydration.ts

```diff
@@ -904,7 +904,7 @@ function toStyleMap(str: string): Map<string, string> {
     let [key, value] = item.split(':')
     key = key.trim()
     value = value && value.trim()
-    if (key && value) {
+    if (key) {
```

Comment:
the `value` has already been validated using `isRenderableAttrValue` during SSR,